datafusion/core/tests/physical_optimizer/enforce_sorting.rs

```diff
@@ -193,19 +289,28 @@ async fn test_remove_unnecessary_sort5() -> Result<()> {
     let join = hash_join_exec(left_input, right_input, on, None, &JoinType::Inner)?;
     let physical_plan = sort_exec([sort_expr("a", &join.schema())].into(), join);
 
+    let test = EnforceSortingTest {
```

Comment:
You can see that the plans did not change by turning on whitespace blind diff: https://github.com/apache/datafusion/pull/17900/files?w=1